from0k2bp.tex

```diff
@@ -1080,73 +1093,80 @@
-& s_y = es + s_d \\
-& t_z = e^2 t + et_1 + t_0 \\
-\end{align*}
-
-First, note that here we are sending the blinded forms $\mathbf{f}_x, \mathbf{f}_y$ of the two
-vectors, not the Pedersen commitments -- the idea is that the Verifier
+data, sent by the Prover, less bewildering\footnote{One may forget what $t$~variable 
+in \eqref{inner_product_response_step_data:5} represents. 
+It comes from $C_z := tH + zG$ defined in \eqref{inner_product_proof_C_xyz_def:1}.}:
+
+\begin{align}
+& \mathbf{f}_x := e\mathbf{x} + \mathbf{d}_x \label{inner_product_response_step_data:1} \\
+& \mathbf{f}_y := e\mathbf{y} + \mathbf{d}_y \label{inner_product_response_step_data:2} \\
+& r_x := er + r_d \label{inner_product_response_step_data:3} \\
+& s_y := es + s_d \label{inner_product_response_step_data:4} \\
+& t_z := e^2 t + et_1 + t_0 \label{inner_product_response_step_data:5}
+\end{align}
+
+First, note that here we are sending the blinded forms $\mathbf{f}_x, \mathbf{f}_y$ in 
+\eqref{inner_product_response_step_data:1} and \eqref{inner_product_response_step_data:2}, 
+not the Pedersen commitments. The idea is that the Verifier
 will verify precisely by reconstructing the commitments and checking
-they match $C_x, C_y$. Those two checks are:
-\begin{align*}
-& eC_x + A_d =? \ r_x H + \mathbf{f}_x \mathbf{G} \\
-& eC_y + B_d =? \ s_y H + \mathbf{f}_y \mathbf{G} \\
-\end{align*}
-The $r_x, s_y$ were chosen to make the random values equate in the relevant
-Pedersen commitments. It's not hard to see that these equations will
+they match $C_x, C_y$. Those two verification checks are:
+\begin{align}
+& eC_x + A_d \stackrel{?}{=} \ r_x H + \mathbf{f}_x \mathbf{G} \label{inner_product_response_step_comms:1} \\
+& eC_y + B_d \stackrel{?}{=} \ s_y H + \mathbf{f}_y \mathbf{G} \label{inner_product_response_step_comms:2}
+\end{align}
+The $r_x, s_y$ were defined in \eqref{inner_product_response_step_data:3} and 
+\eqref{inner_product_response_step_data:4} to make the random values equate 
+in the relevant Pedersen commitments. It's not hard to see that equations 
+\eqref{inner_product_response_step_comms:1}~and~\eqref{inner_product_response_step_comms:2} will
 verify for honest behaviour (remember, this is called ``completeness''):
-\begin{align*}
-& eC_x + A_d \\
-& = e(rH + \mathbf{xG}) + r_d H + \mathbf{d}_x \mathbf{G} \\
-& = (er + r_d) H + \left(e\mathbf{x} + \mathbf{d}_x\right)\mathbf{G} \\
-& = r_x H + \mathbf{f}_x \mathbf{G} \\
-\end{align*}
+\begin{align}
+eC_x + A_d & = e(rH + \mathbf{xG}) + (r_d H + \mathbf{d}_x \mathbf{G}) \quad\text{(See $A_d$ definition \eqref{inner_product_proof_AB_d_def:1})}\\
+           & = (er + r_d) H + \left(e\mathbf{x} + \mathbf{d}_x\right)\mathbf{G}  \\
+           & = r_x H + \mathbf{f}_x \mathbf{G}
+\end{align}
+and similarly for the \textbf{y} equations.
+
+Definition for $t_z$ defined in \eqref{inner_product_response_step_data:5} is needed for the third verification check\footnote{The funny symbol $\because$ means \emph{because}.}:
```

Comment:
You need a definite article: *The* definition for ...
Also the symbol is not "funny" :) it's a standard mathematical symbol; but of course adding that note is doubtless helpful for some readers.

Reply:
"the" added and removed "funny" :)